release/release.go

```diff
@@ -105,15 +103,13 @@ func readOSRelease() OS {
 	return osRelease
 }
 
-var ioutilReadFile = ioutil.ReadFile
+var fileExists = func(path string) bool {
```

Comment:
we have `osutil.FileExists` so just doing what we do for ReadFile except now for FileExists should be fine.

Reply:
I'm aware, this is what I mentioned in the previous comment. I don't know how to get rid of the import cycle generated by importing `osutil`

Reply:
Oh, sorry, silly me!

Reply:
I looked a bit, disentangling the imports is probably not worth it so your approach looks fine - but a comment should be added. Maybe something like: `// Note that osutil.FileExists cannot be used here as an osutil import will create a cyclic import`.

Reply:
That makes perfect sense, thank you!